dpd/src/types.rs

```diff
@@ -77,6 +79,9 @@ pub enum DpdError {
     Transceiver(#[from] TransceiverError),
     #[error("QSFP port \"{qsfp_port}\" has no transceiver")]
     MissingTransceiver { qsfp_port: QsfpPort },
+    /// Usability is currently determined by `check_module_support`.
+    #[error("The transceiver has failed basic operability checks")]
```

Comment:
This is subjective, but I might be more specific about what "unusable" or "operability checks" mean. I would say "Could not identify the transceiver's management interface" or something pretty explicit like that.

Reply:
That information is not available to us by the time we report this error. All we have is a `Transceiver::Unsupported` enum at `transcievers/mod.rs:52. The reason I left the breadcrumb to `check_module_support` in the doc comment is so that it's easily discoverable what checks are in play and what other log messages to look for.